### PR TITLE
[CM-1027] Fixed empty body of notification for rich message in some cases

### PR DESCRIPTION
**Pull request by pahadi777**
Added notification body when the rich message only has button and no message to show .

### CODE REVIEW

File: kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java [hunk 1]
```diff
@@ -51,6 +55,7 @@
  * Time: 7:36 PM
  */
 public class NotificationService {
+    public static final String NOTIFICATION_TEXT_NOT_AVAILABLE = "You received a message";
```
Comment (Sathyan-Elangovan) [inline kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java:58]: Are we using the same in Web?

**Review by Sathyan-Elangovan — CHANGES_REQUESTED** [review]
Add a changelog entry

File: kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java [hunk 2]
```diff
@@ -28,6 +29,7 @@
 import com.applozic.mobicomkit.channel.service.ChannelService;
 import com.applozic.mobicomkit.contact.AppContactService;
 import com.applozic.mobicomkit.listners.AlConstantsHandler;
+import com.applozic.mobicomkit.listners.ApplozicUIListener;
```
Comment (amntoppo) [inline kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java:32]: Remove unused imports

**Comment by Sathyan-Elangovan** [comment]
@pahadi777  Add PR description as well.

File: kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java (hunk 1 above)
Comment (pahadi777) [reply kommunicate/src/main/java/com/applozic/mobicomkit/api/notification/NotificationService.java:58]: No but rajeesh sir told to use this string as default

**Review by amntoppo — APPROVED** [review]

**Comment by pahadi777** [comment]
> Add a changelog entry

@Sathyan-Elangovan Changelog has been added you can check that once

**Review by Sathyan-Elangovan — APPROVED** [review]

**Pull request merged**